packages/api/orm/migration/1638312144983-initial.ts

```diff
@@ -0,0 +1,232 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class initial1638312144983 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const sql = `    
+    SET statement_timeout = 0;
+    SET lock_timeout = 0;
+    SET idle_in_transaction_session_timeout = 0;
+    SET client_encoding = 'UTF8';
+    SET standard_conforming_strings = on;
+    SELECT pg_catalog.set_config('search_path', '', false);
+    SET check_function_bodies = false;
+    SET xmloption = content;
+    SET client_min_messages = warning;
+    SET row_security = off;
```

Comment:
Should these be part of the initial migration? Or the initial bootstrapping? Are these even really necessary?

Reply:
I wonder how many of these are exports of defaults.

Reply:
Probably most? Certainly the two `SET`s I highlighted later on are defaults. So I get the feeling a lot of these may be too.

Reply:
I've looked each setting in docs and found they are either already defaults or settings used by _pg_dump_.  Will remove all.

Reply:
Removed in a08ce9d6ff25b8e25ad86c2b481b7c68b7c6a28b